### DISCUSSION TITLE
[Question]: Hailo-8L support means also suport Hailo-8?

### DISCUSSION THREAD

**Discussion by MungGoon** [blakeblackshear/frigate#13565] [Ask A Question] [answered]
### What is your question?

Hi all,

I saw there is a code to support Hailo-8l.

I'm just curious if it also means supporting Hailo-8, too.

**Comment by NickM-27** [comment]
I would think so but difficult to say for sure

**Reply by sairon** [reply]
FWIW, Hailo-8 (on Raspberry Pi AI HAT+ 26 TOPS) works fine with the same stack (i.e. driver, firmware and userspace) that's used for H8L. It just needs a model compiled for this device, otherwise the performance would be degraded (I haven't tested it but it's what the warning said).

**Comment by OmriAx** [comment, marked as answer]
Hey @MungGoon,
Omri here from Hailo. I'm an application engineer and community manager.
We're working on adding Hailo support to Frigate. It's gonna build on top of what's already there, making things even better. We're pushing to get it out soon, so keep an eye out!

Let me know if you've got any questions.

Cheers,
Omri

**Reply by aeozyalcin** [reply]
That's great to hear! Can we also expect Hailo-8 PCIE modules to also work on x86/64 systems natively in Frigate, or support will be only for Raspberry Pis?

**Reply by NickM-27** [reply]
amd64 arch is supported as well

**Reply by Jimmi61** [reply]
@OmriAx I wanted to use h8 PCIe card on a minipc, is there any news regarding the development?

**Reply by aeozyalcin** [reply]
It already works! I've been using my Hailo 8 M.2 PCIE module on an Intel mini PC since the beginning of the year!

**Reply by Jimmi61** [reply]
@aeozyalcin it's good, thanks. Is the stable container working or shall I use the -h8l version?

**Reply by aeozyalcin** [reply]
I believe it's still the -h8l version for v15. But v16 main will have support
